examples/workers/kv-todos/model.ts

```diff
@@ -47,6 +47,15 @@ export function find(username: string, uid: string) {
 	return DB.read<Todo>(TODOS, key, 'json');
 }
 
+/**
+ * Checks if a specifc `Todo` record already exists
+ */
+ export async function exists(username: string, uid: string) {
```

Comment:
Please remove this new utility. It's redundant as `find()` serves the same purpose. This PR should only have a fix to `DB.read`

Reply:
I've rebased all changes now into [`aca4266` (#19)](https://github.com/lukeed/worktop/pull/19/commits/aca42663e6cc2a8f8d5cf64082bf7cb775e1df09), it looks like typescript compiler is compiling into syntax which node12 does not support